### PR TITLE
Ready for Code Review

### PR DESCRIPTION
**Pull request by camfilho**
Opening a pull request to check for linters errors.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,830 @@
...
+  .white-nav-1 {
+    display: flex;
+    align-items: center;
+    background-color: white;
+    color: #555;
+    height: 50px;
+    font-size: 20px;
+    justify-content: space-between;
+  }
+}
```
Comment (maelfosso) [inline style.css:1002]: That css file doesn't take in consideration all the home page breakpoints..
Please, be sure to identically represent the original page.

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello coders,

Thank you for that first version :+1: 
However there is a lot that should be done. The main thing to say is that what you did doesn't look like the original one.

- [x] CSS styles should use media queries for 3 breakpoints
- small: max-width around 700
- medium: max-width around 1000
- large: mwidth over 1000
 You should rework by taking that in consideration because it's not what you did.

- [ ] Remove the empty **heading**, I mean empty **h1 to h6**

- [x] Please, put space between the elements in the footer
Screenshot from 2019-10-04 13-07-40

- [ ] The main navigation bar and the second doesn't look identical. Also the logo must appear when we scroll down. Also, correct the appearance at different breakpoints. Also the navigation bar is **fixed**

- [x] **Top 3 articles section** 
Screenshot from 2019-10-04 13-21-51
The original don't have space between and around. Also for small screens, the 3 images take the maximum width and are are aligned in one column.

- [ ] The same errors occurs with most of the section from **Latest news section**

Please kindly make the changes.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job :+1: 
You have made most of the requested changes but forgot one crucial requirement. Please look at my suggestion below;
- [ ] add the TNW logo. This is a must-have requirement.

### Please make that change and send for another review.

**Comment by camfilho** [comment]
@whiz25 could you give us a hint of how to achieve it without using Javascript?

thanks in advance

**Comment by camfilho** [comment]
Added the logo on desktop size.
It appears when we scroll down, as in the original one.

**Review by wathigo — APPROVED** [review]
## Hi @camfilho  and @cerfectus :wave:  
The project looks great and meets all the requirements!

**You may move on** 

_Happy coding :computer:_

**Pull request merged**